src/windows/tile_inspector.c

```diff
@@ -1470,24 +1475,47 @@ static void window_tile_inspector_tool_update(rct_window* w, int widgetIndex, in
 	map_invalidate_selection_rect();
 }
 
-static void window_tile_inspector_tool_down(rct_window* w, int widgetIndex, int x, int y)
-{
+void window_tile_inspector_update_selected_tile(rct_window *w, int x, int y) {
+	//if call matches previous mouse coordinates, do not continue.
+	if (x == windowTileInspectorToolMouseX && y == windowTileInspectorToolMouseY) {
+		return;
+	}
+	windowTileInspectorToolMouseX = x;
+	windowTileInspectorToolMouseY = y;
+
 	short mapX = x;
 	short mapY = y;
 	int direction;
 	screen_pos_to_map_pos(&mapX, &mapY, &direction);
 	if (mapX == (short)0x8000) {
 		return;
 	}
+	//if call matches previously selected item, do not continue.
+	if (mapX == windowTileInspectorToolMapX && mapY == windowTileInspectorToolMapY) {
+		return;
+	}
+	windowTileInspectorToolMapX = mapX;
+	windowTileInspectorToolMapY = mapY;
 
 	windowTileInspectorTileSelected = true;
+
 	windowTileInspectorTileX = mapX >> 5;
 	windowTileInspectorTileY = mapY >> 5;
-
+	
```

Comment:
I use visual studio and sometimes i don't catch them. I wish visual studio highlighted blank lines with whitespace or something, it'd make it easier to catch.